src/test/resources/unrc/dose/Bono.feature

```diff
@@ -0,0 +1,91 @@
+
+Feature: test based challenge equals
+     Scenario Outline: User solves an unsolved test based challenge.
+Given that user <user> is already logged on 
+And the user wants to solve the test based challenge
+And the user has already solve this challenge before with a score of 12 points
+And the text of this challenge is
+'''
+	Public static boolean equals (int a, int b){
+   		If (a != b) {
+     			return true;
+		}
+		else{
+    			return false;
+		} 
+	}
+ '''
+
+And the test of the challenge is 
+''' 
+	@Test
+    		Public
+			void testEquals(){
+   	     		assertEquals(false,equals(2,2));
+    			}
+'''
+  			And the proposed solution is
+'''
+	Public static boolean equals (int a, int b){
+   
+		If (a == b) {
+     			return true;
+		}
+'''
+ProgramRepairGolf-DOSE2019
+
+
+ 
+
+
+
+
+
+
+
+Feature: Recover password
+Scenario: A registered user wants to recover his password
+Given that a user wants to recover his password
```

Comment:
Remember that when writing a gherkin scenario, all steps should be as specific as possible, this means that the `Given` step should talk about a specific user.